application/org.openjdk.jmc.flightrecorder.ui/src/main/java/org/openjdk/jmc/flightrecorder/ui/pages/JavaApplicationPage.java

```diff
@@ -205,6 +206,8 @@ public Color getColor(IQuantity balance) {
 			return null;
 		}, Messages.JavaApplicationPage_COLUMN_THREAD_DURATION,
 				Messages.JavaApplicationPage_COLUMN_THREAD_DURATION_DESC);
+		HISTOGRAM.addPercentageColumn(PROFILING_PERCENTAGE_COL, JdkAggregators.EXECUTION_SAMPLE_COUNT, "Percentage",
+				"Percentage over total Profiling samples");
```

Comment:
This looks to be the only instance in the class that uses non-externalized strings, should these be put into Messages.java?

Reply:
This should have been externalised, Its fixed now, Please re-check and update your comments.